vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/cmd/remote/SshjCommandExecutor.kt

```diff
@@ -0,0 +1,72 @@
+package com.malinskiy.marathon.ios.cmd.remote
+
+import ch.qos.logback.classic.Level
+import com.malinskiy.marathon.log.MarathonLogging
+import net.schmizz.sshj.DefaultConfig
+import net.schmizz.sshj.SSHClient
+import net.schmizz.sshj.common.LoggerFactory
+import net.schmizz.sshj.connection.channel.direct.Session
+import org.slf4j.Logger
+import java.io.File
+import java.net.InetAddress
+import java.util.concurrent.TimeUnit
+
+private const val DEFAULT_PORT = 22
+
+class SshjCommandExecutor(val hostAddress: InetAddress,
+                          val remoteUsername: String,
+                          val remotePrivateKey: File,
+                          val port: Int = DEFAULT_PORT,
+                          verbose: Boolean = false) : CommandExecutor {
+
+    val ssh: SSHClient
+
+    init {
+        val config = DefaultConfig()
+        val loggerFactory = object : LoggerFactory {
```

Comment:
do we need this logic here?

Reply:
with `DEBUG` enabled, sshj gets _very_ noisy; might be revisited in the future